.github/CODEOWNERS

```
@@ -75,8 +75,8 @@
 # @raitobezarius is not "code owner", but is listed here to be notified of changes
 # pertaining to the Nix package manager.
 # i.e. no authority over those files.
-pkgs/tools/package-management/nix/                    @raitobezarius
-nixos/modules/installer/tools/nix-fallback-paths.nix  @raitobezarius
+pkgs/tools/package-management/nix/                    @raitobezarius @NixOS/nix-team
+nixos/modules/installer/tools/nix-fallback-paths.nix  @raitobezarius @NixOS/nix-team
```

Considering the comment above,

```suggestion
pkgs/tools/package-management/nix/                    @NixOS/nix-team @raitobezarius 
nixos/modules/installer/tools/nix-fallback-paths.nix  @NixOS/nix-team @raitobezarius 
```